pegasus/sites.v3/code.org/public/educate/resources/videos.md.erb

```diff
@@ -32,6 +32,18 @@ We are building a growing library of educational videos available for re-use by
     <%=view :display_video_thumbnail, id: "introducing_ai", video_code: "Ok-xpKjKp2g", caption: "Introducing How AI Works", play_button: 'center', letterbox: 'false', facebook: facebook, twitter: twitter %>
   </div>
 
+  <div class="col-50" style="float: left; padding: 10px; margin-bottom: 40px">
```

Comment:
Learning a lot from these column styles!

Reply:
Yeah! The `col-50` class is part of the `brstrap.css` stylesheet. (This is a legacy page though so the padding and margin aren't ideal.)